### PR TITLE
change base docker image to redhat-ubi8

### PR DESCRIPTION
**Pull request by ram-infrac**
**What type of PR is this?**
> enhancement

**What this PR does / why we need it**:
Change docker base image for stork from alpine to UBI-8

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:

```release-note

```

**Does this change need to be cherry-picked to a release branch?**:
yes, 2.4.1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,14 @@
-FROM alpine:latest
+FROM registry.access.redhat.com/ubi8/ubi
 MAINTAINER Portworx Inc. <support@portworx.com>
 
-RUN apk add tar python3 ca-certificates && apk upgrade
+LABEL name="Portworx operator" \
```
Comment (piyush-nimbalkar) [inline Dockerfile:4]: *Stork
update the label values based on stork

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,23 @@
-FROM alpine:latest
+FROM registry.access.redhat.com/ubi8-minimal:latest
 MAINTAINER Portworx Inc. <support@portworx.com>
 
-RUN apk add tar python3 ca-certificates && apk upgrade
+ARG VERSION=master
```
Comment (disrani-px) [inline Dockerfile:4]: Won't this overwrite whatever is passed in from the Makefile?

Comment (ram-infrac) [reply Dockerfile:4]: if VERSION is not passed via build-arg use `master`

File: Dockerfile [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+
+RUN microdnf clean all && microdnf install -y yum
+
+RUN yum install -y wget python3 ca-certificates tar gzip
 
 RUN wget -O /usr/local/bin/aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-07-26/bin/linux/amd64/aws-iam-authenticator && \
     chmod +x /usr/local/bin/aws-iam-authenticator
-RUN pip3 install --upgrade pip && pip3 install awscli && pip3 install rsa --upgrade
+
+RUN python3 -m pip install awscli && python3 -m pip install rsa
```
Comment (disrani-px) [inline Dockerfile:20]: Can you also add the `--upgrade` flag. It'll upgrade the packages to the latest one if already installed.

**Pull request merged**